src/test/scala/no/ndla/articleapi/service/converters/HTMLCleanerTest.scala

```diff
@@ -28,9 +30,76 @@ class HTMLCleanerTest extends UnitSuite {
+
+    val expectedIngressResult = LanguageIngress(Some("Medievanene er i endring."), Some("http://image-api/images/5452"))
+    val expectedIngressResult2 = LanguageIngress(Some("Når man driver med medieproduksjon, er det mye arbeid som må gjøres på dataarbeidsplassen. Det kan bli timer med skriving på tastatur, og enda flere timer med redigering og bruk av datamus. Da er det bra å kunne skifte arbeidsstilling og bruke ulike pekeverktøy."), Some("http://image-api/images/5359"))
+
+    val (result, status) = HTMLCleaner.convert(defaultLanguageContent.copy(content=content2), defaultImportStatus)
+    result.content should equal(expectedContentResult2)
+    result.ingress should equal(Some(expectedIngressResult2))
+  }
+
+  test("ingress text is not extracted when not present") {
+    val content = """<section>
+                    |<figure data-size="fullbredde" data-url="http://image-api/images/5452" data-align="" data-id="1" data-resource="image" data-alt="Mobiltelefon sender SMS"></figure>
+                    |<h2>Mediehverdagen</h2>
+                    |</section>""".stripMargin.replace("\n", "")
+    val expectedContentResult = """<section><h2>Mediehverdagen</h2></section>"""
+    val expectedIngressResult = LanguageIngress(None, Some("http://image-api/images/5452"))
+    val (result, status) = HTMLCleaner.convert(defaultLanguageContent.copy(content=content), defaultImportStatus)
+
+    result.content should equal(expectedContentResult)
+    result.ingress should equal(Some(expectedIngressResult))
+    result.requiredLibraries.length should equal (0)
+  }
+
+  test("ingress image is not extracted when not present") {
+    val content =
+      """<section>
+          |<p><strong>Du har sikkert opplevd rykter og usannheter</strong></p>
+          |<ul>
+          |<li><a href="#" title="Snopes">Snopes</a></li>
+          |</ul>
+        |</section>
+      """.stripMargin.replace("\n", "")
+    val expectedContentResult = """<section><ul><li><a href="#" title="Snopes">Snopes</a></li></ul></section>"""
+    val expectedIngressResult = LanguageIngress(Some("Du har sikkert opplevd rykter og usannheter"), None)
+    val (result, status) = HTMLCleaner.convert(defaultLanguageContent.copy(content=content), defaultImportStatus)
+
+    result.content should equal(expectedContentResult)
+    result.ingress should equal(Some(expectedIngressResult))
+    result.requiredLibraries.length should equal (0)
+  }
+
+  test("ingress text is extracted when wrapped in <strong> tags") {
```

Comment:
`<strong>` er egentlig den eneste måten vi har for å indentifisere at teksten er ingress. Hvis vi prøver å hente ut den første paragrafen som bare er wrappet i `<p>` risikerer vi å hente ut ingress selv for artikler som ikke har ingress.

Måten vi identifiserer ingress på er basically: Første seksjon som har en tekst som er wrappet i `strong` og `p`, eller bare `strong`.